src/Interpreters/join_common.cpp

```diff
@@ -276,6 +276,20 @@ void joinTotals(const Block & totals, const Block & columns_to_add, const Names
     }
 }
 
+void addDefaultValues(MutableColumnPtr & column, const DataTypePtr & typ, size_t count)
```

Comment:
Maybe `type` instead of `typ`? It's just one letter...

Reply:
Ok, I used `typ` because `type` is a keyword in some languages, but C++ it is not the case, so I'll change it to `type`